UPGRADE_3.11.md

```diff
@@ -0,0 +1,82 @@
+## SpoonHTTP is deprecated
+
+As you all know Spoon isn't supported anymore, therefore we are trying to 
+replace all functionality that was provided by Spoon with SYmfony-functionality
+or native code. 
+
+With this upgrade we have replace all functionality provided with SpoonHTTP. 
+There are several methods, each one is discussed below.
+
+### SpoonHTTP::getContent()
+
+Before:
+
+```php
+SpoonHTTP::getContent($url)
+```
+
+After:
+
+```php
+file_get_content($url)
+```
+
+> Remark: fopen wrapper should be enabled.
+
+### SpoonHTTP::redirect()
+
+Before:
+
+```php
+\SpoonHTTP::redirect($url);
+```
+
+After:
+
+```php
+throw new RedirectException(
```

Comment:
Maybe add the $this->redirect option too, because this is the preferred way in action files.